### DISCUSSION TITLE
Implement Steam provider

### DISCUSSION THREAD

**Discussion by CogalTek** [pocketbase/pocketbase#3669] [Q&A] [answered]
Hello,

I'm trying to implement the steam provider in the pocketbase frontend, how should I go about it? I imported the image in svg, I added it in /ui/provider.js

```Javascript
    {
        key:   "steamAuth",
        title: "Steam",
        logo:  "steam.svg",
    },
```

but it is not displayed, why is that?

**Comment by ganigeorgiev** [comment]
If you are not using the vite dev server, the ui/dist assets need to be recreated with `npm run build`.

For more details you could also check [Making changes in the Admin UI](https://github.com/pocketbase/pocketbase/blob/master/CONTRIBUTING.md#making-changes-in-the-admin-ui).

If you haven't already, as a refrence implementation you can use the recent OAuth2 PRs, for example the [Patreon provider](https://github.com/pocketbase/pocketbase/commit/71f9be3cb0d812ac982d502bee5f510cb0355c6a).

**Reply by ganigeorgiev** [reply, marked as answer]
btw. Not sure if this is the right documentation for Steam as I have never used it, but based on [this docs](https://partner.steamgames.com/doc/features/auth#website) it seems that Steam is an OpenID Connect (OIDC) provider, meaning that you may not need a custom OAuth2 implementation and can use the existing generic `oidc` provider (_usually the last ones in the Admin UI auth providers listing_).

**Reply by CogalTek** [reply]
ok thank you

**Comment by ritker** [comment]
did you manage to make it work?

**Reply by CogalTek** [reply]
nop sorry
